src/Microsoft.DotNet.Tools.Publish/Program.cs

```diff
@@ -50,18 +54,58 @@ public static int Main(string[] args)
                     path = Directory.GetCurrentDirectory();
                 }
 
-                // Load project context and publish it
-                var fx = NuGetFramework.Parse(framework.Value());
-                var rids = new[] { runtime.Value() };
-                var context = ProjectContext.Create(path, fx, rids);
+                // get the lock file
+                var projectDir = path.ToLower().EndsWith(Project.FileName) ?
```

Comment:
Instead of having the comment above, extract this into a method called GetProjectDirectory?